gc/base/GCExtensionsBase.hpp

```diff
@@ -570,6 +573,7 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	float excessiveGCFreeSizeRatio;
 
 	MM_Heap* heap;
+	MM_SparseVirtualMemory *largeObjectVirtualMemory;
```

Comment:
you can add something like:
large virtual memory for large objects, sparsely committed only for live ones